go/worker/storage/committee/checkpoint_sync.go

```diff
@@ -385,20 +375,19 @@ func (n *Node) checkCheckpointUsable(cp *checkpoint.Metadata, remainingMask outs
 	return maskNone
 }
 
-func (n *Node) updateLastSynced(check *checkpoint.Metadata, mask outstandingMask) {
+func (n *Node) updateLastSynced(syncState watcherState) {
 	n.syncedLock.Lock()
 	defer n.syncedLock.Unlock()
 
-	n.syncedState.LastBlock.Round = check.Root.Version
-	n.syncedState.LastBlock.Namespace = check.Root.Namespace
-	if mask == maskIO {
-		n.syncedState.LastBlock.IORoot = check.Root
-	} else if mask == maskState {
-		n.syncedState.LastBlock.StateRoot = check.Root
-	}
+	n.syncedState = syncState
```

Comment:
This does not seem to update the persistent state (see `case finalized := <-n.finalizeCh` in `node.go`)?